src/boxed/uint/add.rs

```diff
@@ -27,6 +27,8 @@ impl CheckedAdd<&BoxedUint> for BoxedUint {
 
 #[cfg(test)]
 mod tests {
+    #![allow(clippy::unwrap_used)]
```

Comment:
Perhaps this way instead? (I find it a bit unusual to use `#!` on an inline module)

```suggestion
#[cfg(test)]
#[allow(clippy::unwrap_used)]
mod tests {
```

Reply:
I guess I wanted to create more of an "internal property" feeling. But no problem, I'll change it.